php/Plugin.php

```diff
@@ -92,7 +93,13 @@ public function require_deprecated() {
 	 * Requires helper functions.
 	 */
 	private function require_helpers() {
-		require_once __DIR__ . '/Helpers.php';
 		require_once __DIR__ . '/BlockApi.php';
+
+		if ( function_exists( 'block_field' ) || function_exists( 'block_row' ) ) {
+			return;
```

Comment:
This looks to be needed because in the BL to GCB migration, BL is active first, will have loaded these functions.

Then, it tries to activate this plugin, which can cause an error be redefining these functions.